pkgs/development/tools/ocaml/ocamlformat/generic.nix

```diff
@@ -62,6 +63,7 @@ buildDunePackage {
     then [
       base
       (if lib.versionAtLeast version "0.21.0" then cmdliner_1_1 else cmdliner_1_0)
+      csexp # lib.versionAtLeast version "0.22.4"
```

Comment:
Why is this a comment?

Reply:
This is only needed for the `0.22.4` version and we couldn't find a way of adding it only at this specific version without copying and pasting the whole dependency list.

What do you think?